client-app/pages/confirm-email.vue

```diff
@@ -1,25 +1,42 @@
 <template>
   <!-- Success -->
-  <VcEmptyPage v-if="success" image="/static/images/email.webp" class="flex grow flex-col">
+  <VcEmptyPage v-if="emailConfirmed || linkSent" image="/static/images/email.webp" class="flex grow flex-col">
     <template #description>
-      <h1 v-t="'pages.confirm_email.header'" class="mb-8 text-center text-3xl font-bold uppercase lg:text-left" />
+      <h1 class="mb-8 text-center text-3xl font-bold uppercase lg:text-left">
+        {{ $t("pages.confirm_email.header") }}
+      </h1>
 
       <div class="mb-10 flex flex-col items-center gap-5 lg:flex-row">
-        <span
-          class="flex h-12 w-12 shrink-0 items-center justify-center rounded-full bg-[color:var(--color-success)] text-white"
-        >
-          <i class="fa fa-check text-2xl" />
-        </span>
+        <VcIcon name="check" size="xxl" class="text-[color:var(--color-success)]" />
 
         <p class="max-w-md text-center text-19 lg:text-left">
-          <strong v-t="'pages.confirm_email.subtitle'" class="mb-2 block" />
-          {{ $t("pages.confirm_email.text") }}
+          <span v-if="linkSent">
+            {{ $t("pages.confirm_email.link_sent_text") }}
+          </span>
+
+          <span v-else-if="emailConfirmed">
+            <strong class="mb-2 block">
+              {{ $t("pages.confirm_email.subtitle") }}
+            </strong>
+
+            {{ $t("pages.confirm_email.text") }}
+          </span>
         </p>
       </div>
     </template>
 
     <template #actions>
-      <VcButton :to="{ name: 'SignIn' }" class="w-36">
+      <VcButton v-if="linkSent" :to="{ name: 'Home' }" class="w-36">
+        <span v-if="isAuthenticated">
+          {{ $t("common.links.dashboard") }}
```

Comment:
isAuthenticated make separate btn

Reply:
I think it's better to change acceptance criteria about that and make it the same as in all other places: just Home.